### PR TITLE
DASH-130 Add Admin Partner profile creation page

### PR DESCRIPTION
**Pull request by vmkarpenko**

### CODE REVIEW

File: client/src/pages/AdminPartnersListPage/components/updatePartner/validation.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { AnyObject, RefProfile } from '../../../../types';
+
+export const validate = (values: RefProfile): AnyObject => {
```
Comment (trukhilio) [inline client/src/pages/AdminPartnersListPage/components/updatePartner/validation.ts:3]: Could you use `'@lemoncode/fonk'` validation here?

File: client/src/redux/admin/actions.ts [hunk 2]
```diff
@@ -133,3 +133,15 @@ export const getFioAccountsProfilesList = (
   ],
   promise: (api: Api) => api.admin.fioAccountsProfilesList(limit, offset),
 });
+
+export const GET_PARTNERS_REQUEST = `${prefix}/GET_PARTNERS_REQUEST`;
+export const GET_PARTNERS_SUCCESS = `${prefix}/GET_PARTNERS_SUCCESS`;
+export const GET_PARTNERS_FAILURE = `${prefix}/GET_PARTNERS_FAILURE`;
+
+export const getPartnersList = (
+  limit = 25,
```
Comment (trukhilio) [inline client/src/redux/admin/actions.ts:142]: We have a couple constants with `25` value. Let's create one constant `DEFAULT_LIMIT` for client and server and use it everywhere in the app.
Let's do the same for `DEFAULT_OFFSET` too

**Review by trukhilio — CHANGES_REQUESTED** [review]
Check comments

File: client/src/pages/AdminPartnersListPage/components/updatePartner/validation.ts (hunk 1 above)
Comment (vmkarpenko) [reply client/src/pages/AdminPartnersListPage/components/updatePartner/validation.ts:3]: I can’t find a way how to validate list of domains, so I used custom validation function

**Review by trukhilio — APPROVED** [review]
Check one more comment

**Pull request merged**